app/views/users/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Users</h1>
+
+  <% @users.each do |user| %>
+    <%= user.name %>
+    <%= user.email %>
+    <br>
+  <% end %>
+<%== pagy_nav(@pagy) %>
```

Comment:
what is the use of == ?

Reply:
It is assuming the incoming html is safe.
We can also write <%= pagy_nav(@pagy).html_safe %>